src/hotspot/share/logging/logConfiguration.hpp

```diff
@@ -123,6 +124,11 @@ class LogConfiguration : public AllStatic {
 
   // Rotates all LogOutput
   static void rotate_all_outputs();
+
+  static bool is_async_mode() { return _async_mode; }
+  static void set_async_mode(bool value) {
+    _async_mode = value;
+  }
```

Comment:
8229517: missing changes in [logDecorators.hpp](https://github.com/openjdk/jdk/commit/41185d38f21e448370433f7e4f1633777cab6170#diff-d2a0b96dcf40197d99b4d0c795d2f9802aa5a4de4a9a7a1b85666d687b8a783e)